From eaa3e4f83584c1c9cf2dc9d69453c1be186ca84b Mon Sep 17 00:00:00 2001
From: khalvat-M <mmba313@gmail.com>
Date: Fri, 5 Apr 2024 21:31:31 +0330
Subject: [PATCH 1/3] Set PRODUCT_COMPRESSED_APEX := false

for some reason we can not disable it from device side

Change-Id: Ie8fb10dac3de0436e549fe7e91c586ccd8104da5
---
 target/product/updatable_apex.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/target/product/updatable_apex.mk b/target/product/updatable_apex.mk
index 8357fdf..952769f 100644
--- a/target/product/updatable_apex.mk
+++ b/target/product/updatable_apex.mk
@@ -23,4 +23,4 @@ PRODUCT_SYSTEM_PROPERTIES := ro.apex.updatable=true
 # Note: this doesn't mean that all pre-installed apexes will be compressed.
 #  Whether an apex is compressed or not is controlled at apex Soong module
 #  via compresible property.
-PRODUCT_COMPRESSED_APEX := true
+PRODUCT_COMPRESSED_APEX := false
-- 
2.45.2

